ansible_rulebook/validators.py

```diff
@@ -0,0 +1,31 @@
+import json
+import logging
+import os
+from typing import ClassVar, Dict, List
+
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+
+DEFAULT_RULES_SCHEMA = "../schema/ruleset_schema.json"
+logger = logging.getLogger(__name__)
+
+
+class Validate:
+    schema: ClassVar[Dict] = None
+
+    @classmethod
+    def _get_schema(cls):
+        if cls.schema is None:
+            package_dir = os.path.dirname(os.path.realpath(__file__))
+            schema_file = os.path.join(package_dir, DEFAULT_RULES_SCHEMA)
```

Comment:
Is preferable to use the builtin `resources` library instead. Example: https://github.com/ansible/eda-server/blob/375aaddca1f9928a4105708f05f5cf5ddc226076/src/eda_server/config/__init__.py#L39

Reply:
@Alex-Izquierdo Changed to use resources, I am not sure of there is a well known place to store json schemas in python packages.

Reply:
Good point. I think it should be inside the package. It needs a `manifest` to include the file in the distributed package: https://packaging.python.org/en/latest/guides/using-manifest-in/